web/src/app/users/SendTestDialog.tsx

```diff
@@ -57,34 +59,33 @@ export default function SendTestDialog(
   const [now] = useState(DateTime.utc())
   const status = data?.userContactMethod?.lastTestMessageState?.status ?? ''
   const cmDestValue = data?.userContactMethod?.formattedValue ?? ''
-  const cmType: ContactMethodType = data?.userContactMethod?.type ?? ''
-  const lastTestVerifyAt = data?.userContactMethod?.lastTestVerifyAt ?? ''
-  const timeSinceLastVerified = now.diff(DateTime.fromISO(lastTestVerifyAt))
+  const cmType: ContactMethodType | '' = data?.userContactMethod.type ?? ''
```

Comment:
```suggestion
  const cmType: ContactMethodType | '' = data?.userContactMethod?.type ?? ''
```
In typescript, optional chaining needs to continue to the end